orion_mcp.py

```
@@ -902,9 +1168,116 @@ def _timestamp_after(timestamp_val, cutoff_datetime: datetime) -> bool:
+        prior_result = await run_orion(config=full_path, version=version, lookback=str(lookback * 2), input_vars=iv)
+        prior_sum_raw = await summarize_result(prior_result)
+        if isinstance(prior_sum_raw, dict):
+            for run in prior_sum_raw.get("runs", []):
+                run_ts = run.get("timestamp", cutoff_ts)
+                if isinstance(run_ts, (int, float)) and run_ts >= cutoff_ts:
+                    continue
+                for m_name, m_data in run.get("metrics", {}).items():
+                    v = m_data.get("value")
+                    if v is not None:
+                        prior_sum.setdefault(m_name, []).append(v)
+    except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)